tasks/config_jsonsettings.yml

```diff
@@ -0,0 +1,6 @@
+- name: Create .env.json
+  template: src=.env.json.j2 dest={{ django_path }}/.env.json
```

Comment:
You don't need to use templating if the contents of your dest file is in a variable. You can use the `content` option of `copy`: http://docs.ansible.com/ansible/copy_module.html